### PR TITLE
db expression Now() support

### PR DESCRIPTION
**Pull request by lestatcheb**
we need support of Now() db expression

### CODE REVIEW

File: tests/test_field_now.py [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+    assert tm.get_dirty_fields() == {'datetime_field': None}
+
+
+@pytest.mark.django_db
+def test_dirty_fields_field_now2():
+    # we can't compare Now() values, so just test keys
+
+    tm = TestDatetimeModelStandard.objects.create(datetime_field=Now())
+
+    assert set(tm.get_dirty_fields().keys()) == {'datetime_field'}
```
Comment (romgar) [inline tests/test_field_now.py]: This is not a correct assertion for the general `django-dirtyfields` approach: if the model has been just created, none of its fields should be dirty,

Comment (lestatcheb) [reply tests/test_field_now.py]: Seems that without full refactoring with __setattr__ or something like that we can't do this.
Sorry to bother, we can close this pull request.

**Comment by LincolnPuzey** [comment]
@lestatcheb I will close this PR as you suggested. However I think your first test case is valid and dirty-fields should be fixed to pass it. I have opened issue #176 to track it.

File: tests/test_field_now.py (hunk 1 above)
Comment (miratcan) [reply tests/test_field_now.py:26]: Are there any fix for that feature?

File: src/dirtyfields/dirtyfields.py [hunk 2]
```diff
@@ -59,12 +60,16 @@ def _as_dict(self, check_relationship, include_primary_key=True):
 
             # If current field value is an expression, we are not evaluating it
             if isinstance(field_value, (BaseExpression, Combinable)):
-                continue
+                # support only Now() expression
+                if isinstance(field_value, Now):
```
Comment (miratcan) [inline src/dirtyfields/dirtyfields.py:64]: This can be written like this:

```
if not is instance(field_value, Now):
    continue
```

**Pull request closed without merging**